src/Templates/NServiceBusWindowsService/EndpointConfigurationExtensions.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Diagnostics;
+using NServiceBus;
+
+namespace NServiceBusWindowsService
+{
+    // TODO: move this into a shared project for use in all endpoints
+    internal static class EndpointConfigurationExtensions
```

Comment:
I kind of feel like this is going a bit too far with the template.

Reply:
Agreed, IMO we should remove it

Reply:
My personal preference is to keep it. It leads the user down the right path for when they add the next project to the solution, and it makes the `Host.Start()`  code read very nicely.